os/kernel/init/os_bringup.c

```diff
@@ -115,6 +115,15 @@
 #endif
 #endif
 
+#if defined(CONFIG_EXAMPLES_ELF)
```

Comment:
Please don't add example code in kernel.

Reply:
Do you want to remove it now ? or shall we remove it once the binary manager is ready?
If we remove it from this patch, I dont know how to run the apps during app start.

Reply:
elf launch is moved to preapp start from os bringup.